addons/web/static/src/js/views/kanban/kanban_view.js

```diff
@@ -70,16 +71,17 @@ var KanbanView = BasicView.extend({
             editable: activeActions.edit,
             deletable: activeActions.delete,
             read_only_mode: params.readOnlyMode,
+            selectionMode: params.selectionMode,
         };
         this.rendererParams.quickCreateEnabled = this._isQuickCreateEnabled();
-        this.rendererParams.readOnlyMode = params.readOnlyMode;
+        this.rendererParams.selectionMode = params.selectionMode;
         var examples = archAttrs.examples;
         if (examples) {
             this.rendererParams.examples = kanbanExamplesRegistry.get(examples);
         }
 
         this.controllerParams.on_create = archAttrs.on_create;
-        this.controllerParams.hasButtons = true;
+        this.controllerParams.readOnlyMode = false;
```

Comment:
Why this line has been deleted?
Why we have to force readOnlyMode to false?

Reply:
@adr-odoo It was mistake, I removed readOnlyMode line and hasButtons should based on selectionMode, we do not want to show Create button in Selection Kanban in header